examples/elmoControl/include/ElmoProtocol.h

```diff
@@ -0,0 +1,53 @@
+#ifndef KICKCAT_CAN_ELMO_PROTOCOL_H
+#define KICKCAT_CAN_ELMO_PROTOCOL_H
+
+namespace kickcat
+{
+    struct SDOEntry
+    {
+        uint16_t index;         ///< Main index in the SDO dict
+        uint8_t  subindex;      ///< Subindex in the SDO dict
+    };
+
+    namespace sdo
+    {
+        constexpr SDOEntry RxPDO { 0x1C12, 0 };
```

Comment:
Init here with values from protocol.h `CoE::SM_CHANNEL + 2` (fell free to add SM_CHANNEL_0 etc)

Reply:
BTW not sure if this abstraction if of any use: may be more useful to have the constant directly in the main.cc 

```
bus.writeSDO(bus.slaves().at(0), 0x1C12, 0, Bus::Access::COMPLETE, 
     const_cast<uint8_t*>(pdo::RxMapping), rxLength);
```